egs/TAC/local/conf.yml

```diff
@@ -14,7 +14,6 @@ net:
   n_src: 2
   window_ms: 4
   context_ms: 16
-  sample_rate: 16000
```

Comment:
If it's not passed to the model, the loading will be wrong, right?

Reply:
If it is specified here then there is a conflict with the command line one and the recipe fails.

Reply:
Okay i will change train.py to take sample_rate from the json